### PR TITLE
Fix paging example

### LINKED ISSUE #406: Fix Paging example
The current Paging example is not correct, because the .paging() function returns a VecDeque of pages and not a VecDeque of User.

The provided code prints something like that : 
```
[
    Response {
        status: 200,
        version: HTTP/1.1,
        headers: {
             //truncated 
        },
        body: Foo {
            id: None,
            user_principal_name: None,
        },
    },
    Response {
        status: 200,
        version: HTTP/1.1,
        headers: {
            //truncated 
        },
        body: Foo {
            id: None,
            user_principal_name: None,
        },
    },
    Response {
        status: 200,
        version: HTTP/1.1,
        headers: {
            //truncated 
        },
        body: Foo {
            id: None,
            user_principal_name: None,
        },
    },
]
```

### PR DESCRIPTION
**Pull request by DevLazio**
Fix #406

### CODE REVIEW

File: examples/paging.rs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+use graph_rs_sdk::*;
+use serde::Deserialize;
+use serde::Serialize;
```
Comment (sreeise) [inline examples/paging.rs:3]:
```suggestion
#[macro_use]
extern crate serde;

use graph_rs_sdk::*;
```

**Review by sreeise — CHANGES_REQUESTED** [review]
One small change. Everything else looks good. Thanks for the PR fix.

**Comment by sreeise** [comment]
If you want to. You can change the examples/next_link directory to examples/paging and add your example in there. Thats what that directory was meant to be. But I just forgot with so many changes that went into 1.0.0

File: examples/paging.rs (hunk 1 above)
Comment (sreeise) [reply examples/paging.rs:3]: Gonna go ahead and commit this and get the example merged in.

**Comment by DevLazio** [comment]
Sorry for the delay, was busy this morning. I see you merged it already, thanks.

**Comment by sreeise** [comment]
> 

No problem at all. Your all good.  I just wanted to go ahead and get the next version update in so thats why I merged it. Thank you for the fix 🚀

**Pull request merged**